### PR TITLE
Add support for near-mock-vm

### PR DESCRIPTION
**Pull request by willemneal**
- Return outcome, receipts
- Add standalone binary for running wasm generated by as.

### CODE REVIEW

**Comment by willemneal** [comment]
@nearmax It's failing the example test since the abort feature isn't on for non-relase test, but I'm not sure what changed where that would matter?  Could it be the removal of the "not wasm target"s?

**Comment by willemneal** [comment]
@evgenykuzyakov Yes, I'm finally moving this to the AS sdk as it's used in unit testing. However, there were still a couple of changes to this sdk that were required.

**Review by frol — APPROVED** [review]
I cannot reason enough about the implications, but the code looks good to me

File: near-sdk-macros/res/near_blockchain.rs [hunk 1]
```diff
@@ -105,6 +107,8 @@ pub mod near_blockchain {
             sys::log_utf16(len, ptr)
         }
 
+        unsafe fn abort(&self, msg_ptr: u32, filename_ptr: u32, line: u32, col: u32) {}
```
Comment (evgenykuzyakov) [inline near-sdk-macros/res/near_blockchain.rs:110]: Is there `sys::abort` ?

**Review by evgenykuzyakov — APPROVED** [review]

File: near-sdk-macros/res/near_blockchain.rs (hunk 1 above)
Comment (willemneal) [reply near-sdk-macros/res/near_blockchain.rs:110]: Not sure what you mean.  This function is present in `near-vm-logic` so it should be mocked here as well and is used by errors in AS.

**Pull request merged**